webapp/src/monaco.tsx

```diff
@@ -1000,9 +1000,7 @@ export class Editor extends toolboxeditor.ToolboxEditor {
                 keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.Enter],
                 keybindingContext: "!editorReadonly",
                 precondition: "!editorReadonly",
-                contextMenuGroupId: "0_pxtnavigation",
```

Comment:
Is this something that should be removed for all editors? It seems like it works well for Arcade and micro:bit.

Reply:
it's not very useful, so we decided to remove it entirely